client/client_test.go

```diff
@@ -540,6 +540,47 @@ func TestHandleModifyResponse(t *testing.T) {
 			},
 			ClientError: "received a session parameter result when there was none pending",
 		}},
+	}, {
+		desc: "AckType set to FIB_ACK, response should contains both FIB and RIB",
```

Comment:
I think we probably should not assume that all vendors will reply both RIB_ACK and FIB_ACK in the case of AckType=FIB_ACK. (some vendor might only return FIB_ACK). 

So, maybe we should change the logic to something like: `AckType set to FIB_ACK, do not delete the pending op until receive a FIB_ACK`.

Reply:
To clarify I meant testing the following scenarios in the case of ActType=FIB_ACK: 
* received AFTResult_FIB_PROGRAMMED, clear the queue. 
* received AFTResult_FIB_FAILED, clear the queue. 
* received AFTResult_FAILED, clear the queue. 

And then in the case of ActType=RIB_ACK
* received AFTResult_RIB_PROGRAMMED, clear the queue.

Reply:
did you mean three test case for three scenario if the ack type set to FIB_ACK ?

Reply:
added